docs/content/dagster-cloud/deployment/serverless.mdx

```diff
@@ -199,6 +199,62 @@ To build and upload the image, use the command line:
        --base-image-tag sha256_518ad2f92b078c63c60e89f0310f13f19d3a1c7ea9e1976d67d59fcb7040d0d6
      ```
 
+### Disabling PEX-based Deploys
+
+Prior to using PEX files, Dagster Cloud deployed code using Docker images. This feature is still available. To deploy using a Docker image instead of PEX:
+
+- **With GitHub**: Delete the `ENABLE_FAST_DEPLOYS: 'true'` line in your GitHub Actions configuration (usually at `.github/workflows/deploy.yml`):
+
+  ```yaml
+  env:
+    DAGSTER_CLOUD_URL: ...
+    DAGSTER_CLOUD_API_TOKEN: ...
+    # ENABLE_FAST_DEPLOYS: 'true' # disabled
+  ```
+
+- **With the CLI**: Use the `deploy` command instead of the `deploy-python-executable` command:
+
+  ```shell
+  dagster-cloud serverless deploy \
+    --location-name example \
+    --package-name assets_modern_data_stack
+  ```
+
+The Docker image deployed can be customized as below.
```

Comment:
```suggestion
The Docker image deployed can be customized using either lifecycle hooks or customizing the base image.
```

Or something along these lines - I just want to tie the options we're outlining to this sentence.